src/org/dotnet/ci/triggers/GenericTriggerBuilder.groovy

```diff
@@ -7,7 +7,8 @@ import jobs.generation.Utilities
 // Periodic - Periodic triggers against a Git repo
 class GenericTriggerBuilder implements TriggerBuilder {
```

Comment:
Yeah, interfaces are possible (see TriggerBuilder.groovy).

I don't see other types coming in here.